### PR TITLE
[ADD] Manufacturing: Production analysis

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR adds a new doc detailing how to use the production analysis report in the Manufacturing app.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @Felicious,

This doc is ready for the first round of review.

Thanks,
John

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+To open the production analysis report, navigate to :menuselection:`Manufacturing app --> Reporting
+--> Production Analysis`.
+
+Search... bar
+=============
+
+The :guilabel:`Search...` bar appears at the top of the :guilabel:`Production Analysis` dashboard.
+Along with its drop-down menu, the :guilabel:`Search...` bar used to refine the statistics shown on
+the production analysis report.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:20]: This, perhaps, is just a difference in how we present information. I personally would be a little confused and think the search bar is for searching for keywords or information.

Instead, I'd name this section something like "Filters and groups"

Below is an optional rewording suggestion to focus on what the report is used for.
```suggestion
Filters and groups
=============

In the :guilabel:`Search...` bar at the top of the :guilabel:`Production Analysis` dashboard, various
selections of filters and groupbys are available to refine the statistics and show different information in
the production analysis report.
```

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+By default, the :guilabel:`Search...` bar includes the :guilabel:`End Date: [current year]` filter,
+so that data is only displayed for |MOs| completed within the current calendar year. To remove this
+filter, and show older data as well, click the :icon:`oi-close` :guilabel:`(Remove)` button on the
+right side of the filter.
+
+To only show the data for a specific product, type the product's name in the :guilabel:`Search...`
+bar, and press enter. Multiple products can be entered at the same time, so that only the data for
+those products is shown.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:29]: What do you mean by multiple products can be entered at the same time? 
image

Do you mean that multiple products can appear in the search results? As it's written, I think I'm interpreting the information as: multiple searches for products can be entered in the search bar

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+Filters
+-------
+
+The :guilabel:`Filters` section of the drop-down menu is used to select the time period for which
+production data is displayed. To select a time period, click the :icon:`fa-caret-down`
+:guilabel:`(toggle)` button next to :guilabel:`End Date`, and click one of the options.
+
+The time periods to choose from are the three most recent months, each quarter of the current fiscal
+year (:guilabel:`Q1`, :guilabel:`Q2`, :guilabel:`Q3`, :guilabel:`Q4`), and the three most recent
+years.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:50]: Possibly add a note that additional filter options can be found by clicking the Add Custom Filter option.

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+specific categories. This makes each graph type show data based on that category, rather than
+individual weeks.
+
+.. example::
+   With the bar chart graph type selected, clicking the :guilabel:`Product` option in the
+   :guilabel:`Group By` section causes the chart to show one bar for each product, instead of one
+   bar for each week during the selected time period.
+
+Comparison
+----------
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:65]: Although the pie chart example you showed is quite useful, I personally think comparing between periods is also cool in the pivot table. Perhaps you can include both examples in the doc? 

image

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+
+*Measures* are the datasets that can be selected in the production analysis report. Each dataset
+represents a specific statistic about |MOs| in the database. Choose a measure by clicking the
+:guilabel:`Measures` :icon:`fa-caret-down` button, and selecting one of the following options from
+the drop-down menu:
+
+- :guilabel:`Average Component Cost/Unit`: the average cost of the components required to produce
+  one unit of the product.
+- :guilabel:`Average Employee Cost/Unit`: the average cost paid to employees to produce one unit of
+  the product.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:118]:
```suggestion
- :guilabel:`Average Employee Cost/Unit`: the manual labor cost to produce one unit, calculated using the employee's hourly rate and the time spent on operations
```

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+- :guilabel:`Average Subcontracting Cost/Unit`: the average cost of engaging a subcontractor to
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:119]: engaging?

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst [hunk 7]
```diff
@@ -0,0 +1,318 @@
...
+
+Use case: compare products
+==========================
+
+One of the best uses for the production analysis report is comparing statistics about two or more
+products. This is accomplished by entering the products into the :guilabel:`Search...` bar, then
+selecting the necessary measure, filter, and grouping, to see the desired data.
+
+.. example::
+   Toy manufacturer *Tommy's Toys* is trying to reduce their manufacturing operation costs. To
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:291]: Interesting example written in narrative format!

**Review by Felicious — APPROVED** [review]
Everything looks good to me, @hojo-odoo ! Great job with this reporting doc :blush:

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst (hunk 4 above)
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:65]: Thanks for pointing this out. I've updated the copy to clarify that the pivot view can be used as well.

File: content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst (hunk 5 above)
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:118]: This is not necessarily true. There are other ways that Odoo computes employee cost, depending on certain configuration options. I think it's a bit too much to explain on this one bullet point though.

**Pull request closed without merging**